blog/2024/09-10-release-roadmap-10.10.0/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Release Roadmap for 10.10.0
+description: We are about to start the 10.10.0 release cycle; here is the roadmap. Please help us test!
+authors: joshuaboniface
+date: 2024-09-10
```

Comment:
Date should be omitted as it is already in the folder structure

Reply:
Are you sure? I've had date on all the previous ones, but if it's "safe" that's fine.

Reply:
The frontmatter takes priority over the folder name, but it's easier to stick to just one way to set the date so they don't get out of sync when a blog date changes before publishing.
Our previous site generator required the date in frontmatter.